src/sumOfOther.js

```diff
@@ -0,0 +1,13 @@
+module.exports = function sumOfOther(arr) {
+  const res = [];
+  for (let i = 0; i < arr.length; i += 1) {
+    let tmp = 0;
+    for (let j = 0; j < arr.length; j += 1) {
+      if (i !== j) {
+        tmp += arr[j];
+      }
+    }
+    res.push(tmp);
+  }
+  return res;
+};
```

Comment:
Small recommendation, let's separate the parts of the code with line breaks, how did you do in function recursion, 
It will make it easier to read